### PR TITLE
Best practices and recommendations from audit

### PR DESCRIPTION
**Pull request by kantp**

### CODE REVIEW

File: documentation/deploy.md [hunk 1]
```diff
@@ -19,29 +21,35 @@ If you want to change your admin contract, you can write a contract that `extend
 
 [!NOTE] Note that if you want to use a custom admin contract, you should write the admin contract
 from scratch. Inheriting from `FungibleTokenAdmin` and overwriting specific methods might not work.
-You can find an example of a custom admin contract in `FungibleToken.test.ts`.
+You can find an example of a custom admin contract in `FungibleToken.test.ts`.ß
```
Comment (ymekuria) [inline documentation/deploy.md:24]:
```suggestion
You can find an example of a custom admin contract in `FungibleToken.test.ts`.
```

Comment (ymekuria) [reply documentation/deploy.md:24]: I think this was a mistaken character.

File: FungibleToken.ts [hunk 2]
```diff
@@ -60,24 +51,29 @@ export class FungibleToken extends TokenContract {
 
   async deploy(props: FungibleTokenDeployProps) {
     await super.deploy(props)
-
-    this.admin.set(props.admin)
-    this.decimals.set(props.decimals)
-    this.paused.set(Bool(false))
-
-    this.account.tokenSymbol.set(props.symbol)
+    this.paused.set(Bool(true))
```
Comment (ymekuria) [inline FungibleToken.ts:54]: I'm just curious. Do you set the paused flag in the initialized method as well as here in deploy?

**Review by ymekuria — APPROVED** [review]
LGTM! I just had some questions and minor comments.

File: documentation/deploy.md (hunk 1 above)
Comment (kantp) [reply documentation/deploy.md]: Yes, indeed. Thanks for spotting!

File: FungibleToken.ts (hunk 2 above)
Comment (kantp) [reply FungibleToken.ts:54]: It's probably superfluous. We urge people to use an atomic deploy, calling the `deploy()` functions of the admin and token contract, as well as `initialize()` all in one transaction. But if they don't, we at least want no transactions to happen before `initialise()` is called.

**Pull request merged**